.github/workflows/labeler.yml

```diff
@@ -10,7 +10,7 @@ permissions:
 jobs:
   triage:
     if: github.actor != 'discourse-translator-bot'
-    runs-on: debian-12
```

Comment:
This is just a PR labeler, no need for our own hosted runners which is overkill.